packages/wrangler/src/api/dev.ts

```diff
@@ -84,13 +82,13 @@ export interface UnstableDevOptions {
 		devEnv?: boolean;
 		fileBasedRegistry?: boolean;
 		vectorizeBindToProd?: boolean;
+		enableIpc?: boolean;
```

Comment:
This is a new flag—it's required to make the Pages use of `unstable_dev` work with the ipc channel setup for fixture tests.